src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -80,6 +81,30 @@ static int uct_cuda_ipc_iface_is_reachable(const uct_iface_h tl_iface,
             *((const uint64_t *)dev_addr)) && ((getpid() != *(pid_t *)iface_addr)));
 }
 
+static double uct_cuda_ipc_iface_get_bw()
+{
+    double bw = 6911 * 1024.0 * 1024.0;
+    CUdevice_attribute attrib = CU_DEVICE_ATTRIBUTE_COMPUTE_CAPABILITY_MAJOR;
+    CUdevice cu_device;
+    int count;
+    int major_version;
+
+    UCT_CUDADRV_FUNC(cuDeviceGetCount(&count));
+    ucs_assert(count > 0);
+
+    UCT_CUDADRV_FUNC(cuDeviceGet(&cu_device, 0));
+    UCT_CUDADRV_FUNC(cuDeviceGetAttribute(&major_version, attrib, cu_device));
```

Comment:
do we need check status here? Will `major_version` be undefined if `cuDeviceGetAttribute` fails?

Reply:
Fixed. Thanks